pkg/graveler/graveler.go

```diff
@@ -2674,6 +2684,20 @@ func (g *Graveler) CherryPick(ctx context.Context, repository *RepositoryRecord,
 	return commitID, nil
 }
 
+func applyCommitOverrides(commit *Commit, commitOverrides CommitOverrides) {
+	if commitOverrides.Message != "" {
+		commit.Message = commitOverrides.Message
+	}
```

Comment:
What about empty message? We allow it for commits with a flag (`allow-empty-message`)

Reply:
we allow it on the CLI level, graveler is not aware of it and can accept empty strings as commit messages AFAIK.